source/_patterns/04-components/breadcrumb/breadcrumb.twig

```diff
@@ -3,9 +3,11 @@
   modifier_classes ? modifier_classes,
 ]|join(' ')|trim %}
 
-<nav aria-labelled="breadcrumb-label" class="{{ classes }}" role="navigation">
+{% set breadcrumb_label = 'breadcrumb-label'|unique_id %}
```

Comment:
I think we should rename this variable to `breadcrumb_id` to make its purpose clear.